### ISSUE TITLE
Relatório da Votação - resultado falho

### ISSUE THREAD

**Issue by jpmehl** [redelivre/delibera#94]
No relatório de votação, mesmo votando em mais de uma proposta, só apareceu uma com votação.

**Comment by lpirola** [comment]
@jacsonp tem ideia do q pode ser isso?

**Comment by jacsonp** [comment]
@jpmehl pode explicar melhor, mandar links, etc, veja que no exemplo a seguir está aparecendo corretamente, embora poderia ter gráficos e um desenho melhor:
http://novocnpc.beta.redelivre.org.br/pauta/priorizacao-das-metas-do-setorial-de-danca/

**Comment by jpmehl** [comment]
Sim, para além de ter que melhorar a apresenaçào, coloca pra "imprimir" um relatório e vai ver que não tá computando nada.
